### ISSUE TITLE
No crew when boarding enemy ships

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#268]
Boarded enemy ships in space no longer have crew or misc objectives, landed enemy ships no longer drop off NPCs.

---

## Details 
When boarding ships, the misc quest (kill crew or leave ship) no longer starts, no enemies spawn, and nothing spawns in the captain's locker. I'm assuming the quest handles the crew and loot spawns, so something is wrong. The quest ID is 98b4e.

From my own experience and from others who are experiencing this bug, the boarding quest works properly for non hostile ships that you're boarding/pirating. The quest is called 'BE_Objective', so I'm assuming BE stands for board enemy.

It seems to be permanently linked to a ship I had previously boarded, hard to say exactly when, but I can go back to a save from before the bug started and the boarding quest will trigger properly.

Possibly related, I can now enter landed Starborn ships on planets. And also, hostile ships landed on planets (starborn included) no longer drop off NPCs. I've seen other people with the same issue also report this, so it's probably related.

## Game Version
12212450

## New Game
No

## Reproduction Steps
I am not completely sure, but I've seen others say it started for them by boarding an enemy ship (either in space or on the ground), not killing everyone, and then leaving without taking the ship.

**Comment by Bobbyclue** [comment]
This quest seems to lack any kind of shutdown procedure for if you leave without claiming the ship. I'd recommend adding the player to an alias on the quest with a script attached that stops the quest when leaving the ship. These changes will require the CK or xEdit.

**Comment by Constellation-VASCO** [comment]
Can be fixed with these two commands:

    resetquest 2c0fd9

    resetquest 98b4e

I knew about 98b4e, but a mod to fix this specific bug also mentions 2c0fd9, not sure what quest that is, but running those two commands together works perfectly.

**Comment by Constellation-VASCO** [comment]
Would using these commands disable achievements?

**Comment by Constellation-VASCO** [comment]
Please fix! Hitting the same issue. Talking about taking a lot of fun out of the game...

**Comment by Constellation-VASCO** [comment]
wskeever created a mod for this issue. There is also some excellent instructions for further troubleshooting and debugging this issue in the comments section for the mod:

https://www.nexusmods.com/starfield/mods/5668

**Comment by kayahr** [comment]
I encountered this bug as well and investigated it for some hours. For me the problem was the **BE_Generic** (_2c0fd9_) quest, not **BE_Operative**. Resetting this quest helps for a short time but the bug will reappear very quickly again. The reason for this is that there are additional 25 quests (which I consider to be a quest pool): **BE_Generic_00** up to **BE_Generic_24**. Ships in space and landed ships fetches a free quest from this pool (or the **BE_Generic** quest itself) and starts it. The game is supposed to complete/reset these quests when a ship is captured, destroyed or leaves or the player leaves. But sometimes this does not happen. And after it happened too often the quest pool is full and the ships cannot start one of these quests any more. This causes the empty ships. This also causes Starborn ships to be accessible because the **BE_Generic** quests are also responsible for making landed Starborn ships inaccessible.

To check if you are affected by this bug, do this:

1. Go to space and fly to a planet where you are alone (no other ships in orbit).
2. Run these commands to output the quest status of all the **BE_Generic** quests:

    ```
    sqs 002c0fd9
    sqs 001ef7a7
    sqs 001ef7a8
    sqs 001ef7a9
    sqs 001ef7aa
    sqs 001ef7ab
    sqs 001ef7ac
    sqs 001ef7ad
    sqs 001ef7ae
    sqs 001ef7af
    sqs 001ef7b0
    sqs 001ef7b1
    sqs 001ef7b2
    sqs 001ef7b3
    sqs 001ef7b4
    sqs 001ef7b5
    sqs 001ef7b6
    sqs 001ef7b7
    sqs 001ef7b8
    sqs 001ef7b9
    sqs 001ef7ba
    sqs 001ef7bb
    sqs 001ef7bc
    sqs 001ef7bd
    sqs 001ef7be
    sqs 001ef7bf
    ```

If my understanding of the quests is correct then they all should be in stopped state. If they are all running then you are affected by the bug and you can fix it by resetting all these quests with these commands:

```
resetquest 002c0fd9
resetquest 001ef7a7
resetquest 001ef7a8
resetquest 001ef7a9
resetquest 001ef7aa
resetquest 001ef7ab
resetquest 001ef7ac
resetquest 001ef7ad
resetquest 001ef7ae
resetquest 001ef7af
resetquest 001ef7b0
resetquest 001ef7b1
resetquest 001ef7b2
resetquest 001ef7b3
resetquest 001ef7b4
resetquest 001ef7b5
resetquest 001ef7b6
resetquest 001ef7b7
resetquest 001ef7b8
resetquest 001ef7b9
resetquest 001ef7ba
resetquest 001ef7bb
resetquest 001ef7bc
resetquest 001ef7bd
resetquest 001ef7be
resetquest 001ef7bf
```

One cause of this bug which I can 100% reproduce is the landing of Starborn ships. Whenever a starborn ship lands in my vicinity one of these quests is started and never stopped. This happens a lot in late game when exploring planets so I usually reset these quests when I start or exit the game so I have no problems during a game session.

I tried reproducing it with pirate ships, boarding it, not boarding it and leaving it behind, leaving the planet, waiting for it to launch again. In all these situations the quest was properly stopped. So maybe fixing this quest script for Starborn ships would be enough to fix the bug.

There are more quests with **BE_** prefix, so maybe special ships have similar special quests. And I have no idea what the **BE_Objective** quest has to do with it. I never had to reset this quest so far. Maybe it is related to ships connected to a mission board quest? I don't know.

**Comment by kayahr** [comment]
Oh, I just noticed that the mod from wskeever mentioned above is not the mod with the simple bat script resetting the BE_Generic script. This mod actually contains quest script fixes. I'll try it out later, maybe this indeed fixes the issue for good.

**Comment by Constellation-VASCO** [comment]
Checked out wskeever's mod and it worked, nice! Now Starborn ships land, drop of some soldiers and launch again and the quest is stopped. Never knew that is how Starborn ships should actually work. It even stopped older quests which were still running in the pool. So after I witnessed a Starborn ship landing and launch the quest pool is now empty again. No more quest resetting needed for now. At least not for this issue.
